src/framework/theme/components/super-search/styles/super-search.component.column-curtain.scss

```diff
@@ -0,0 +1,157 @@
+/**
```

Comment:
Generally, regarding the styles, please consider changing the architecture to the one used in the other components. We can discuss this on Wednesday.